Update FAQ section (#316)
todogroup/governance: TODO-OSPO-Ambassador-Program.md

* Receive a TODO OSPO Ambassadors Credly badge issued by the Linux Foundation

## 🙋‍♀️ Frequent Asked Questions:

### What is the process for becoming an Ambassador?
To become an Ambassador, an individual must review and meet the current requirements. We invite you to [submit your application for consideration](https://github.com/todogroup/governance/issues/new/choose) if you meet the requirements.

### Is the TODO OSPO Ambassador (TOAs) a paid position?
No, the TODO OSPO Ambassador is not a paid position. However, TOAs will receive rewards like limited edition swag and exclusive discounts to LF events for their service.

### How does changing organizations or ceasing activities within an organization affect Ambassador status?

> The Ambassador title is granted to an individual, regardless of their affiliation. Thus, moving to a different organization or ceasing activities in a given organization does not affect the Ambassador's title unless the individual determines that this shift will affect their responsibilities and believes they can no longer commit to their duties.

Reviewer: Just a suggestion for a slight rephrasing / simplification of language.

```suggestion
The Ambassador title is granted to an individual, regardless of their affiliation. Thus, moving to a different organization or ceasing activities in a given organization does not affect the Ambassador's title unless the individual determines to no longer be able to commit to the duties of a TODO OSPO Ambassador.
```
